### PR TITLE
Feature/cxspa 4235 - cypress tests seller edit

### PR DESCRIPTION
**Pull request by Stofftiger**

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/quote/quote.e2e-flaky.cy.ts [hunk 1]
```diff
@@ -137,4 +139,31 @@ context('Quote', () => {
       quote.checkQuoteState(quote.STATUS_CANCELED);
     });
   });
+
+  describe('Edit quote process - sales reporter perspective (CXSPA-4235)', () => {
```
Comment (Uli-Tiger) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/quote/quote.e2e-flaky.cy.ts]:
```suggestion
  describe('Edit quote process - seller (sales assistant) perspective (CXSPA-4235)', () => {
```

File: projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts [hunk 2]
```diff
@@ -641,14 +644,238 @@ export function gotToQuoteDetailsOverviewPage() {
...
+export function enableEditQuoteMode() {
+  log('Enables the edit mode for the quote', enableEditQuoteMode.name);
+  cy.get('cx-quote-actions-by-role button.btn-secondary').click();
+}
+
+/**
+ * Creates an expiry date for the quote (2 months and 2 days in the future from today)
+ * @returns Expiry date for the quote
+ */
+function createExpiryDate(): Date {
```
Comment (Uli-Tiger) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts]:
```suggestion
function createExpiryDateInNearFuture(): Date {
```

**Review by Uli-Tiger — CHANGES_REQUESTED** [review]
lgtm - just 2 remarks regarding naming

File: projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts (hunk 2 above)
Comment (Uli-Tiger) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts]: or maybe createValidExpiryDate

Comment (Stofftiger) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts]: renamed it to createValidExpiryDate

**Pull request merged**